Code/Editor/EditorPreferencesPageViewportCamera.cpp

```diff
@@ -81,6 +81,7 @@ void CEditorPreferencesPage_ViewportCamera::CameraMovementSettings::Reflect(AZ::
         ->Field("DefaultOrbitDistance", &CameraMovementSettings::m_defaultOrbitDistance)
         ->Field("SpeedScale", &CameraMovementSettings::m_speedScale)
         ->Field("GoToPositionInstantly", &CameraMovementSettings::m_goToPositionInstantly)
+        ->Field("GoToPositionDuration", &CameraMovementSettings::m_gotoPositionDuration)
```

Comment:
Nit: Seems like this should be "m_goToPositionDuration" (capital T) for consistency with m_goToPositionInstantly.

Reply:
Good spot, will fix (fixed up a few other places too)